common/transferimage.h

```diff
@@ -51,6 +54,7 @@ class TransferImage
 
 private:
     QImage m_image;
+    bool m_flipped;
```

Comment:
Instead of the flip bit, could we make this a little bit more generic by using a QTransform instead? That would already pave the way for only sending a sub-rect of the frame for example?